filter.js

```diff
@@ -241,17 +261,17 @@ function printAllChallenges() {
 function filterFunctionSearchBar() {
     const findThis = filterSearchBar.value.toLowerCase().split(" ");
     console.log(findThis)
-    testbox.innerHTML= ""
+    testbox.innerHTML = ""
     fullApiJson.forEach(challenge => {
-        if(findThis.some(test => (test != "") && (challenge.title.toLowerCase().includes(test) || challenge.description.toLowerCase().includes(test)))){
+        if (findThis.some(test => (test != "") && (challenge.title.toLowerCase().includes(test) || challenge.description.toLowerCase().includes(test)))) {
             // console.log(challenge.title.toLowerCase().replaceAll(" ", "").includes(findThis.toLowerCase()))
             const challengeBox = createChallengeBox(challenge)
             testbox.appendChild(challengeBox);
 
-        } else if (findThis[0] == ''){
+        } else if (findThis[0] == '') {
             printAllChallenges()
-        } else 
-        console.log('nope')
+        } else
+            console.log('nope')
```

Comment:
This is risky, and many (most?) programmers avoid this type of syntax where curly braces have been omitted in a conditional statement. Some reasons why: https://stackoverflow.com/questions/2125066/is-it-a-bad-practice-to-use-an-if-statement-without-curly-braces